configlib-core/src/test/java/de/exlll/configlib/SerializersTest.java

```diff
@@ -29,6 +29,15 @@
 import static org.hamcrest.Matchers.*;
 
 class SerializersTest {
+
+    private final String tmpConfigPath = TestUtils.createPlatformSpecificFilePath("/tmp/config.yml");
+    /*
+        If found a way to implement the newline char test on the file name without
+         giving errors on windows. The following field ought to be changed to do so.
+     */
+    private final String tmpWithUnderscorePath = TestUtils.createPlatformSpecificFilePath("/tmp/with_underscore.yml");
+    private final String tmpPath = TestUtils.createPlatformSpecificFilePath("/tmp");
```

Comment:
As these fields are constants, they should be `static` and written in uppercase. The comment should also be deleted. 

Thoughts?